core/src/main/scala/kafka/security/auth/Authorizer.scala

```diff
@@ -22,21 +22,23 @@ import org.apache.kafka.common.Configurable
 import org.apache.kafka.common.security.auth.KafkaPrincipal
 
 /**
- * Top level interface that all plugable authorizer must implement. Kafka server will read "authorizer.class" config
- * value at startup time, create an instance of the specified class and call initialize method.
- * authorizer.class must be a class that implements this interface.
- * If authorizer.class has no value specified no authorization will be performed.
+ * Top level interface that all pluggable authorizers must implement. Kafka will read the "authorizer.class.name" config
+ * value at startup time, and create an instance of the specified class using the default constructor. The class must
+ * implement this interface.
  *
- * From that point onwards, every client request will first be routed to authorize method and the request will only be
- * authorized if the method returns true.
+ * From that point onwards, every client request will first be routed to the authorize method and the request will only
+ * be authorized if the method returns true.
+ *
+ * If authorizer.class.name has no value specified, then no authorization will be performed, and all operations are
+ * permitted.
```

Comment:
It would be good to mention that the `configure` method will be invoked during broker start-up. It was there previously, but it incorrectly said `initialize`.